config/environments/production.rb

```diff
@@ -52,30 +70,34 @@
   # Prepend all log lines with the following tags.
   config.log_tags = [ :uuid, :host ]
 
-  # Use a different logger for distributed setups.
-  # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
-
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = 'http://assets.example.com'
-
+  # Use a real queuing backend for Active Job (and separate queues per environment).
+  # config.active_job.queue_adapter     = :resque
+  # config.active_job.queue_name_prefix = "gibct_data_service_production"
```

Comment:
and more

Reply:
It's part of what got added when running bin/rails app:update. We're not using but I thought they should be left in.